test/sphere_metric_terms.jl

```diff
@@ -18,3 +17,33 @@ using Test
 
     @test sum(ones(space)) ≈ 4pi * radius^2 rtol = 1e-3
 end
+
+@testset "Volume of a sphere" begin
+    FT = Float64
+    radius = FT(3)
+    zlim = (0, 1)
+    helem = 4
+    zelem = 10
+    Nq = 4
+
+    vertdomain = Domains.IntervalDomain(
+        Geometry.ZPoint{FT}(zlim[1]),
+        Geometry.ZPoint{FT}(zlim[2]);
+        boundary_tags = (:bottom, :top),
+    )
+    vertmesh = Meshes.IntervalMesh(vertdomain, nelems = zelem)
+    vert_center_space = Spaces.CenterFiniteDifferenceSpace(vertmesh)
+
+    horzdomain = Domains.SphereDomain(radius)
+    horzmesh = Meshes.Mesh2D(horzdomain, Meshes.EquiangularSphereWarp(), helem)
+    horztopology = Topologies.Grid2DTopology(horzmesh)
+    quad = Spaces.Quadratures.GLL{Nq}()
+    horzspace = Spaces.SpectralElementSpace2D(horztopology, quad)
+
+    hv_center_space =
+        Spaces.ExtrudedFiniteDifferenceSpace(horzspace, vert_center_space)
+    hv_face_space = Spaces.FaceExtrudedFiniteDifferenceSpace(hv_center_space)
+
+    @test sum(ones(hv_center_space)) ≈ 4pi * radius^2 * (zlim[2] - zlim[1]) rtol =
+        1e-3
```

Comment:
Add a comment to explain the formula
```suggestion
    # "shallow" spherical shell: volume = surface area * height
    @test sum(ones(hv_center_space)) ≈ 4pi * radius^2 * (zlim[2] - zlim[1]) rtol =
        1e-3
```

Reply:
"hollow sphere" or just "spherical shell"?

Reply:
I think "spherical shell" is ok: https://en.wikipedia.org/wiki/Spherical_shell

Reply:
Yes, but it is more appropriate to specify thin in this case, otherwise the formula for its volume is different. Thanks

Reply:
You mean "thin" instead of "shallow"? We use shallow atmosphere and deep atmosphere assumptions a lot in the literature. But if you think "thin" is better I can change it.

Reply:
Yes, I think "shallow atmosphere" and "deep atmosphere" correspond to what wikipedia calls thin.

Reply:
I'll stick to it and use `"shallow atmosphere" spherical shell` then. That would be more clear to atmosphere science people I think.